app/src/main/java/com/xmartlabs/scasas/geoquiz/ui/QuizActivity.java

```diff
@@ -70,45 +54,162 @@ public boolean onOptionsItemSelected(MenuItem item) {
   }
 
   private void setupQuizButton() {
-    trueButon = (Button) findViewById(R.id.true_button);
-    trueButon.setOnClickListener(new View.OnClickListener() {
+    addQuestion();
```

Comment:
I thought in that name but I'm afraid that maybe someone could get confuse because of the other method that I called setupQuestion